webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  entry: './build/app/app.js',
+  output: {
+    filename: './build/public/js/app.bundle.js'
+  },
+  module: {
+    preLoaders: [],
+    loaders: [
+      // {test: /\.jsx?$/, exclude: /node_modules/, loader: 'babel'}
```

Comment:
@jsainthilaire @kdemoya 
I did not add babel-loader right now - since we will have the need to run babel-cli to transform the files that run on server-side, we can just create the webpack bundle after babel-cli has finished.
What I am trying to achieve is to create more consistency between server-client by doing the transformations with a single tool.